apps/emqx/src/emqx_persistent_session_ds.erl

```diff
@@ -46,6 +46,7 @@
 
 %% FIXME
 -define(DS_SHARD, <<"local">>).
+-define(DEFAULT_KEYSPACE, <<"#">>).
```

Comment:
Nit. I get the idea, yet at the moment I feel that we'll have some atom names for any keyspaces in the future config anyway. Maybe just?
```suggestion
-define(DEFAULT_KEYSPACE, default).
```

Reply:
Sure.  It wasn't clear to me what we would use for keyspace identifiers, so I assumed they would be topic filters.